homeassistant/components/advantage_air/services.yaml

```
@@ -0,0 +1,9 @@
+set_time_to:
+  description: Sets Time To On/Off
+  fields:
+    entity_id:
+      description: Time To sensor entity
```

I think the description needs a bit more polish, what does it really do? :)
```suggestion
  description: Sets time to On/Off
  fields:
    entity_id:
      description: Time to sensor entity
```

---

So it's the timers for turning the system on and the system off after a set number of minutes. I would welcome your suggestion.

---

Write something like that; controls timers to set system state to on or off?